scipy/sparse/linalg/tests/test_interface.py

```diff
@@ -524,3 +525,13 @@ def test_sparse_matmat_exception():
         A @ np.identity(4)
     with assert_raises(ValueError):
         np.identity(4) @ A
+
+
+@pytest.mark.skipif(IS_PYPY, reason="Test not meaningful on PyPy")
```

Comment:
I wonder if it might be slightly preferable to use `HAS_REFCOUNT` from `numpy.testing`:

`@pytest.mark.skipif(not HAS_REFCOUNT, reason="Python lacks refcounts")`

perhaps slightly more general--I've seen it used a bit more recently

Reply:
Possibly - I was doing it this way for consistency with the existing `assert_deallocated` tests.